### ISSUE TITLE
babelHelper is not defined is ES version

### ISSUE THREAD

**Issue by afitiskin** [bvaughn/react-select-fast-filter-options#10]
When using module with webpack2 (that uses ES version by default) you will get error: 
```
Uncaught ReferenceError: babelHelpers is not defined
```

I found that compiled ES version of the module includes this part of code:
```javascript
if ((typeof _ret === 'undefined' ? 'undefined' : babelHelpers.typeof(_ret)) === "object") return _ret.v;
```
but `babelHelpers` is not included in result compiled bundle.

Current workaround for me is to use:
```javascript
import createFilterOptions from 'react-select-fast-filter-options/dist/commonjs';
```

**Comment by bvaughn** [comment]
If you'd like to submit a PR that fixes this I'll be happy to review it.

**Comment by afitiskin** [comment]
@bvaughn done, please check it

**Comment by bvaughn** [comment]
Fixed via b89e17d, available in 0.2.2
